lib/synchronized_model/publish_mixin.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
```

Comment:
What is the purpose of this method over just using model.attributes in message? 
Is there a possibility it won't just be attributes in the future?

Reply:
Same question as @storres-westernmilling

Reply:
We override `to_message_payload` in some cases. 

https://github.com/westernmilling/item_master/blob/1b61b09d0345b734a66c6a61a5103727ea7d3e5b/app/models/item.rb#L44-L46

So my thinking was to always have `to_message_payload` defined this way and then just overwrite it as needed.

However doing something like `additional_attributes_to_publish` which gets merged with `attributes` in `to_message_payload` is probably cleaner.

 I'll try that unless there's a different specific suggestion.